modules/imaging_browser/php/imaging_browser.class.inc

```diff
@@ -61,6 +61,7 @@ class Imaging_Browser extends \NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        $this->skipTemplate = true;
```

Comment:
i think to be consistent with other modules, you should replace this with `public $skipTemplate = true;` towards the top of the class definition

Reply:
Done!